assemble/bin/accumulo-cluster

```diff
@@ -590,7 +626,22 @@ function main() {
   accumulo_cmd="${bin}/accumulo"
   SSH='ssh -qnf -o ConnectTimeout=2'
 
-  case "$1" in
+  # Copy input arguments into new array
+  # removing any options
+  DEBUG=""
+  i=0
+  declare -a program_args
+  for arg in "$@"; do
+    if [[ $arg == "--dry-run" ]]; then
+      DEBUG="true"
+    else
+      program_args[i++]="$arg"
+    fi
+  done
```

Comment:
This parsing becomes a lot simpler if the dry-run flag is expected to be first. Then, you can just use `shift` to strip it off the front.

If the case block below were put into its own function, it would also simplify the use of the `program_args` array, allowing you to work directly with `$@`, `$1`, `$2`, etc. inside the called method.

Reply:
I left as is. I'm trying to allow `--dry-run` to be placed anywhere in the input.